sql/02-create-table-friend.sql

```diff
@@ -0,0 +1,6 @@
+DROP IF EXISTS friend
+
+CREATE TABLE friend (
+  id1 int4 REFERENCES student(id),
```

Comment:
`INT4` and `INTEGER` are aliases, so you should be consistent of the usage of either across the code.